crates/nu-command/src/env/with_env.rs

```diff
@@ -99,7 +99,7 @@ fn with_env(
                         return Err(ShellError::CantConvert(
                             "string list or single row".into(),
                             x.get_type().to_string(),
-                            call.positional[1].span,
+                            call.positional_iter().nth(1).unwrap().span,
```

Comment:
Yeah, I have to look at the code again to figure out why it's safe. Accessing `Vec` elements with indexing is also essentially the same, right? An out of bounds access would panic in much the same way that `vec.get(n).unwrap()` would.

Reply:
you can add as JT mentioned the `expect` with a message indicating why it should not fail. In case it fails and we get a panic we will know where to look at. No need to exhaust all possibilities why it could panic now since it hasnt panic accessing by index

Reply:
Done, I think the expect messages should be correct